src/common/api/collectionsApi.ts

```diff
@@ -177,7 +177,7 @@ export type ColumnMeta = {
   display_name?: string;
 } & (
   | {
-      type: 'date' | 'int' | 'float';
+      type: 'date' | 'int' | 'float' | 'bool';
```

Comment:
this was a mistake oops
```suggestion
      type: 'date' | 'int' | 'float';
```